app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "About") %>
+<h1><%= t ".title"%></h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em><%= t ".tutorial"%></em></a> <%= t ".is"%>
+  <a href="https://www.railstutorial.org/book"><%= t ".book"%></a> and
+  <a href="http://screencasts.railstutorial.org/"><%= t ".series" %></a>
+  <%= t ".web" %>
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>.
```

Comment:
Sử dụng `link_to` là tag của rails thay vì `a` của html nhé

Sử dụng I18n cho toàn bộ file html.erb em nhé
=> self check nhé

`"%>` sau `"` thì có space nà, em self check lại nhé